components/core/Component/Overlay/Overlay.razor.cs

```diff
@@ -54,6 +54,14 @@ public sealed partial class Overlay : AntDomComponentBase
         [Parameter]
         public int HorizontalOffset { get; set; } = 4;
 
+        /// <summary>
```

Comment:
@mutouzdl for your consideration.

Reply:
I think that's a pretty clear explanation